fdbserver/workloads/ClogRemoteTLog.actor.cpp

```diff
@@ -251,7 +251,7 @@ struct ClogRemoteTLog : TestWorkload {
 				std::vector<std::pair<StorageServerInterface, ProcessClass>> results =
 				    wait(NativeAPI::getServerListAndProcessClasses(&tr));
 				for (auto& [ssi, p] : results) {
-					if (ssi.locality.dcId().present() && ssi.locality.dcId().get() == g_simulator->remoteDcId) {
+					if (ssi.locality.dcId().present() && ssi.locality.dcId().get() == g_simulator->remoteDcId.get()) {
```

Comment:
Did Joshua pass? I remember there are cases where `remoteDcId` is not set. We can do this instead:
```suggestion
					if (ssi.locality.dcId().present() && g_simulator->remoteDcId.present() && ssi.locality.dcId().get() == g_simulator->remoteDcId.get()) {
```